packages/test-utils/src/find.js

```diff
@@ -10,6 +10,26 @@ import {
 import { throwError } from 'shared/util'
 import { matches } from './matches'
 
+/**
+ * Traverses a vue instance for its parents and returns them in an array format
+ * @param {Component} vm
+ * @returns {Component[]} The component and its corresponding parents, in order from left to right
+ */
+export function findAllParentInstances(childVm: any) {
+  const instances = [childVm]
+
+  function getParent(_vm) {
```

Comment:
Any ideas on how to avoid the closure here?

Reply:
This is to find the highest `Vue` with the custom `errorHandler` right? I somehow thought it would "just work" and we wouldn't need to reverse-traverse all the way up. I don't have another way to do this. 

I think having the conditional is fine, no problem from my point of view

Reply:
@lmiller1990 correct. I thought it would work straight out of the box as well, but not exactly for reasons I am having trouble remembering. I want to say it has to do with how the vue instances behave on the component, but don't take my word for it 😅